### PR TITLE
add workflow recipe

### PR DESCRIPTION
**Pull request by tacman**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: survos/workflow-extension-bundle/1.1/config/packages/survos_workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+survos_workflow:
+    direction: LR
+    entities: null
```
Comment (Nyholm) [inline survos/workflow-extension-bundle/1.1/config/packages/survos_workflow.yaml]: You can remove `null`.

**Comment by tacman** [comment]
Thanks.  Should it be

    entities:

or

    entities: ~

Thanks.

**Comment by Nyholm** [comment]
Just having `entities:` is fine

**Comment by Nyholm** [comment]
Friendly ping

**Comment by Nyholm** [comment]
Im closing this due inactivity. Feel free to ping me if you want it reopened.

**Comment by tacman** [comment]
Sorry, if there's a PR I'll accept it, I didn't realize there was.  Thanks.

**Comment by tacman** [comment]
image

**Comment by tacman** [comment]
Is there a button somewhere that says "Accept PR"?

**Comment by Nyholm** [comment]
It seams like you are removing old tags. See issue here: https://flex.symfony.com/r/github.com/symfony/recipes-contrib/792

**Comment by tacman** [comment]
Yes, I did.

Unfortunately, I don't understand github and PR's enough to know what to
do.  This (and landing-bundle) are my first two attempts at writing a
public bundle.  I find the recipes particularly confusing.  (And Ryan
leaves it as an exercise for the reader in his terrific "How to Write A
Third-Party Bundle" symfonycast.

Hope to figure it out soon, thanks for the nudge.

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by Nyholm** [comment]
Friendly ping

**Comment by nicolas-grekas** [comment]
Something went wrong with a merge commit in the history.
Please submit again if you want to have this merged.

**Pull request closed without merging**